application/testing/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # F3D Testing
 function(f3d_test)
 
-  cmake_parse_arguments(F3D_TEST "DEFAULT_LIGHTS;LONG_TIMEOUT;HDRI;INTERACTION;NO_BASELINE;NO_RENDER;NO_OUTPUT;WILL_FAIL" "NAME;CONFIG;DATA;RESOLUTION;THRESHOLD;REGEXP;REGEXP_FAIL;DEPENDS" "ARGS" ${ARGN})
+  cmake_parse_arguments(F3D_TEST "DEFAULT_LIGHTS;LONG_TIMEOUT;HDRI;INTERACTION;NO_BASELINE;NO_RENDER;NO_OUTPUT;WILL_FAIL;NO_FILE_RENDER_TESTING" "NAME;CONFIG;DATA;RESOLUTION;THRESHOLD;REGEXP;REGEXP_FAIL;DEPENDS" "ARGS" ${ARGN})
```

Comment:
what about just `NO_DATA` instead of `NO_FILE_RENDER_TESTING`. Since we specify the data with `DATA`, it makes more sense to me.

Reply:
That doesnt really explain what does the flag do, does it ? One would need to look at the code to understand.

Reply:
NO_DATA_FORCE_RENDER ?

Reply:
From a test dev perspective, you don't really care about the details, it's just calling `f3d_test(DATA foo.bar ...)` or `f3d_test(NO_DATA ...)`.  
So I prefer just `NO_DATA`, but `NO_DATA_FORCE_RENDER` is fine too if you prefer.